cryptus-app/components/wallet_viewer/nft_details/nft_properties/property_block/property_block.module.scss

```diff
@@ -0,0 +1,35 @@
+@use "/styles/variables" as v;
+
+.container {
+    flex: 1;
+    flex-basis: 50%;
+    width: 50%;
+    width: fit-content;
```

Comment:
can't have 2 times the width. Use min-width instead if this what you want to do.